pipeline-examples/push-git-repo/pushGitRepo.groovy

```diff
@@ -15,3 +15,14 @@ sshagent (credentials: ['git-ssh-credentials-ID']) {
     sh("git tag -a some_tag -m 'Jenkins'")
     sh('git push <REPO> --tags')
 }
+
+// For SSH private key authentication and Jenkins Blue Ocean, you can take
+// advantage of the auto generated SSH key...
+
+sshagent (credentials: ['jenkins-generated-ssh-key']) {
+  sh 'git config core.sshCommand "ssh -v -o StrictHostKeyChecking=no"'
```

Comment:
Is `StrictHostKeyChecking=no` really something we want to recommend as a best practice?

Reply:
It is the only way to make it working. I got it from the SSH Agent plugin official homepage. 

Thank you so much